.github/workflows/apidocs-upload.yml

```diff
@@ -0,0 +1,98 @@
+name: apidocs
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+
+  # First check if this version is a valid public release
+  check-version:
+    runs-on: ubuntu-22.04
+    permissions:
+      id-token: write # This is required for requesting the JWT
+    outputs:
+      release_status_output: ${{ steps.release_version.outputs.version_status_output }}
+
+    steps:
+      - name: Check release version
+        id: release_version
+        shell: bash
+        run: |
+  
+          if [[ "${{ github.ref }}" = refs/tags/* ]]; then
+            version="${ref#refs/tags/}"
+
+            # Pattern that accepts "v1.0.0" , "v2.4.6". 
+            if [[ "$version" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+              result=isRelease
+            # Pattern that accepts "v1.0.0-rc1".
+            elif [[ "$version" =~ ^v[0-9]+\.[0-9]+\.[0-9]+-rc[0-9]+$ ]]; then
+              result=isReleaseCandidate
+            else
+              result=isNotRelease
+            fi
+          else
+            result=isNotTag
+          fi
+  
+          echo $result
+          echo "version_status_output=$result" >> $GITHUB_OUTPUT
+
+  # Initiate upload only if the version is valid
+  upload-apidocs:
+    runs-on: ubuntu-22.04
+    needs: check-version
+    env:
+      release_status: ${{ needs.test_version.outputs.release_status_output }}
+    if: ${{ env.release_status == 'isRelease' || env.release_status == 'isReleaseCandidate' }}
+    timeout-minutes: 10
+
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    steps:
+      # https://github.com/actions/checkout/releases/tag/v4.0.0
+      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac
+      - name: Use Node.js ${{ matrix.node-version }}
+        # https://github.com/actions/setup-node/releases/tag/v4.0.0
+        uses: actions/setup-node@8f152de45cc393bb48ce5d89d36b731f54556e65
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install dependencies
+        run: npm install
```

Comment:
We should use `npm ci` for CI runners intead of `npm install`

Reply:
Fixed on [50ca249](https://github.com/HathorNetwork/hathor-wallet-headless/pull/364/commits/50ca249687a0a8f0618b82110084f4ea8ce59c59)